[0.000000] (-) TimerEvent: {}
[0.000263] (-) JobUnselected: {'identifier': 'cartographer_slam'}
[0.000587] (-) JobUnselected: {'identifier': 'localization_server'}
[0.000631] (-) JobUnselected: {'identifier': 'map_server'}
[0.001305] (-) JobUnselected: {'identifier': 'path_planner_server'}
[0.001358] (-) JobUnselected: {'identifier': 'robot_description'}
[0.001382] (-) JobUnselected: {'identifier': 'robot_move'}
[0.001401] (-) JobUnselected: {'identifier': 'sllidar_ros2'}
[0.001422] (full_coverage) JobQueued: {'identifier': 'full_coverage', 'dependencies': OrderedDict()}
[0.001446] (full_coverage) JobStarted: {'identifier': 'full_coverage'}
[0.099779] (-) TimerEvent: {}
[0.200087] (-) TimerEvent: {}
[0.300322] (-) TimerEvent: {}
[0.400556] (-) TimerEvent: {}
[0.500823] (-) TimerEvent: {}
[0.601156] (-) TimerEvent: {}
[0.701447] (-) TimerEvent: {}
[0.801827] (-) TimerEvent: {}
[0.887579] (full_coverage) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 51, in build\n    setup_py_data = get_setup_data(self.context.pkg, env)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/__init__.py", line 20, in get_setup_data\n    return dict(pkg.metadata[key](env))\n                ^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_ros/package_augmentation/ros_ament_python.py", line 57, in getter\n    return get_setup_information(\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 249, in get_setup_information\n    _setup_information_cache[hashable_env] = _get_setup_information(\n                                             ^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 296, in _get_setup_information\n    result = subprocess.run(\n             ^^^^^^^^^^^^^^^\n  File "/usr/lib/python3.12/subprocess.py", line 571, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command \'[\'/usr/bin/python3\', \'-c\', \'import sys;from contextlib import suppress;exec("with suppress(ImportError):    from setuptools.extern.packaging.specifiers    import SpecifierSet");exec("with suppress(ImportError):    from packaging.specifiers import SpecifierSet");from distutils.core import run_setup;dist = run_setup(    \\\'setup.py\\\', script_args=(\\\'--dry-run\\\',), stop_after=\\\'config\\\');skip_keys = (\\\'cmdclass\\\', \\\'distclass\\\', \\\'ext_modules\\\', \\\'metadata\\\');data = {    key: value for key, value in dist.__dict__.items()     if (        not key.startswith(\\\'_\\\') and         not callable(value) and         key not in skip_keys and         key not in dist.display_option_names    )};data[\\\'metadata\\\'] = {    k: v for k, v in dist.metadata.__dict__.items()     if k not in (\\\'license_files\\\', \\\'provides_extras\\\')};sys.stdout.buffer.write(repr(data).encode(\\\'utf-8\\\'))\']\' returned non-zero exit status 1.\n'}
[0.888339] (full_coverage) JobEnded: {'identifier': 'full_coverage', 'rc': 1}
[0.898190] (-) EventReactorShutdown: {}
